### DISCUSSION TITLE
Erweiterung eines bestehenden Systems

### DISCUSSION THREAD

**Discussion by scharrrfi** [hoylabs/OpenDTU-OnBattery#1480] [Q&A]
Hallo zusammen,

ich betreibe aktuell zwei HM-300 an jeweils einer dummen 2,5 kWh 24 V Batterie und einen HM-600 an einer dummen 7,5 kWh  36 V Batteriebank. Das ganze läuft mit einem Skript auf eine ganz passable Nulleinspeisung, sobald die Batterien gebraucht werden und diese werden mit statischen AC-Ladegeräten geladen, sobald genug Überschuss da ist. Als Basis dient eine AhoyDTU.
Vorteile des bisherigem modularen Aufbaus, die Ladegeräte schalten sich je nach Überschuss nach und nach zu bzw. gehen bei nicht mehr genügend Überschuss aus. Auch schalten die Batterien automatisch bei unterschreiten einer bestimmten Spannung bzw. beim Laden ab.

Nun hab ich mir zusätzlich 15 kWh als intelligentes 48 V System geschossen. Die würde ich gerne an einem HM-1500 betreiben und mit einem Huawei R4850G2 dynamisch laden. Mir stellt sich nun die Frage, ob ich das mit openDTUonBattery zusätzlich in meine aktuelle Landschaft integrieren kann oder eine Migration meines bisherigen Systems zu OpenDTUonBattery machen müsste. Die alte Ladesteuerung (über Tasmota Steckdosen) müsste aber vermutlich außerhalb laufen, wenn ich das richtig verstehe.

Vielleicht könnt ihr mir ja ein paar Tipps geben, wie ich meinen Plan realisieren kann. Kann euch gerne auch nochmal entsprechende Skizzen liefern.

Vielen Dank!

**Comment by spcqike** [comment]
Ich denke, du solltest deine externe Steuerung beibehalten und den neuen Speicher mit den 48V Akku genauso einbinden, wie die alten. 

OpenDTU kann aus einem Akku mehrere wechselrichter prima steuern. Und den Akku auch laden. Allerdings kannst du nicht mehrere Akkus einbinden. 

Du brauchst deine vorhandene Steuerung also so oder so noch. Wenn du nun die vorhandene Steuerung und zusätzlich die openDTU für das 48V System betreibst, kann das zu unschönen Schwingungen und übersteuerungen führen. 

Meiner Meinung nach sollte ein zentrales Management System alles verwalten.

**Reply by scharrrfi** [reply]
Danke für die Rückmeldung. Das habe ich mir leider fast gedacht. Leider kann das Skript aktuell nur eine DTU auf einmal ansteuern. Muss mal schauen, wie ich das aufbohren kann bzw. nach einer Alternative umschauen. 

Oder ich binde eben den HM-1500 ebenfalls in Ahoy ein und nutze die OpenDTU on Battery nur zum Steuern des Ladegeräts und auslesen der Batterie. Da kann ich aber sicherlich auch mit ESPHome was schlankeres basteln. 
Eigentlich hat mich OpenDTU on Battery als charmante all in one Lösung angesprochen. Schade, dass es keine unterschiedlichen Batterien managen kann. 
Weiß jemand, ob so etwas für die Zukunft geplant ist?
